### ISSUE TITLE
Server says "(no connection)" on server list page

### ISSUE THREAD

**Issue by ThatOneGuy808** [framewiki/minecraft-server#6]
On the server list page when I add the Framework server, it shows up and gives a description, but it won't let me connect. I have tried on a Steam Deck and a MacBook. I have already confirmed I have typed the server name correctly
<img width="595" alt="image" src="https://user-images.githubusercontent.com/50556872/233800280-de0fb5d6-8f07-41a9-9873-c6a16f6a151f.png">

**Comment by Morpheus636** [comment]
If you PM me your IP address and username via Discord I can check if you're being blocked by something but I think this is an issue with your network environment. 

Check what IP fwmc.morpheus636.com is resolving to. Also make sure you haven't been banned from multiplayer by Mojang.

**Comment by Morpheus636** [comment]
Looks like it's because they're trying to use ipv6. I'll look into fixing but its not a priority.
